src/nvim/testdir/test_expr_utf8.vim

```diff
@@ -0,0 +1,26 @@
+" Tests for expressions using utf-8.
+if !has('multi_byte')
+  finish
+endif
+set encoding=utf-8
```

Comment:
This line should be removed.  Changing encoding isn't supported in nvim anymore.